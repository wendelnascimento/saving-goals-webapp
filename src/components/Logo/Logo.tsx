import React from 'react';

import { Image } from './styles';

import logoSvg from '../../icons/logo.svg';

const Logo = () => <Image src={logoSvg} />;

export default Logo;
