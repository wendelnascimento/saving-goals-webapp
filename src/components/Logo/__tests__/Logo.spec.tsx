import React from 'react';
import { shallow, mount } from 'enzyme';

import Logo from '../Logo';

describe('<Logo />', () => {
  it('Should render', () => {
    shallow(<Logo />);
  });

  it('Should math stored snapshot', () => {
    const wrapper = mount(<Logo />);

    expect(wrapper).toMatchSnapshot();
  });
});
