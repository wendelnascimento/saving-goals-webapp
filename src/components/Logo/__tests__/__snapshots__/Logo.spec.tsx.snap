// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> Should math stored snapshot 1`] = `
.c0 {
  width: 100px;
}

<Logo>
  <styled.img
    src="test-file-stub"
  >
    <img
      className="c0"
      src="test-file-stub"
    />
  </styled.img>
</Logo>
`;
